src/VisualStudio/Core/Def/Implementation/DesignerAttribute/VisualStudioDesignerAttributeService.cs

```diff
@@ -332,17 +331,17 @@ private void AddFilteredInfos(ImmutableArray<DesignerAttributeData> data, ArrayB
         /// <summary>
         /// Callback from the OOP service back into us.
         /// </summary>
-        public Task ReportDesignerAttributeDataAsync(ImmutableArray<DesignerAttributeData> data, CancellationToken cancellationToken)
+        public ValueTask ReportDesignerAttributeDataAsync(ImmutableArray<DesignerAttributeData> data, CancellationToken cancellationToken)
         {
             Contract.ThrowIfNull(_workQueue);
             _workQueue.AddWork(data);
-            return Task.CompletedTask;
+            return new ValueTask();
```

Comment:
```suggestion
            return ValueTask.CompletedTask;
```

Reply:
Unfortunately it's not available on net472 :(